apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx

```diff
@@ -21,8 +22,9 @@ type ContactHistoryItemProps = {
 
 function ContactHistoryItem({ history, setChatId, ...props }: ContactHistoryItemProps): ReactElement {
 	const t = useTranslation();
-	const formatDate = useTimeAgo();
+	const formatDate = useFormatDateAndTime();
 	const username = history.servedBy?.username;
+	const hasClosingMessage = history.closingMessage && history.closingMessage.msg !== '';
```

Comment:
```suggestion
	const hasClosingMessage = history.closingMessage?.msg;
```

Reply:
Why not this?

Reply:
There is a possibility of the room being closed with the closing message as `''` the closing message object will only not exist when a room is closed and the whole comment feature is turned off on the omni configurations

Reply:
how about `str.trim()` ?

Reply:
BTW: I forgot `!!` on the beginning... casting to boolean...

Reply:
<img width="73" alt="image" src="https://user-images.githubusercontent.com/17487063/204696155-71e17993-bf09-45c2-acc0-2b8ccdaf7294.png">

Reply:
how about str.trim() ?